config/routes/speakers.yaml

```diff
@@ -0,0 +1,25 @@
+speaker_index:
+    path: /
+    controller: App\Controller\Speaker\Index::handle
+
+speaker_create:
+    path: /create
+    controller: App\Controller\Speaker\Create::handle
+
+speaker_show:
+    path: /{id}
+    controller: App\Controller\Speaker\Show::handle
+    requirements:
+        id: '\d+'
```

Comment:
IDs should probably be UUIDs rather than auto-increments for security purpose as well as for decoupling the app from the database. UUIDs are not digits, so the constraint here is not good